packages/core/admin/admin/src/content-manager/pages/EditSettingsView/tests/__snapshots__/index.test.js.snap

```diff
@@ -2406,7 +2405,6 @@ exports[`EditSettingsView should add field 1`] = `
                       <span
                         class="c7 c25"
                         for="select-11"
-                        id="select-11-label"
```

Comment:
Mmh, I find the removal of these id attributes a bit concerning because they are being referenced e.g. by `aria-labelledby=` in the same file. Do you have an idea why they are being removed?

Reply:
I just checked how it is handled in the DS side:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/71838159/208619080-a4d38957-3f90-4865-9edf-e551aa736af2.png">

I think we should pass a generatedId to enforce the usage of the same id for the entire `Field`
one issue that I see though (and I don't think it should be fixed in this PR since we have the same issue in the DS component):
`DateTimePicker` renders a Field and generates a `generatedId`, which then contains `DatePicker` and `TimePicker` that both render their own `Field` + `FieldLabel` and their own `generatedId` 
I don't think the label and inputs are linked at the end - wdyt?

Reply:
We've discussed this: the `DateTimePicker` has several problems handling labels. @ronronscelestes will create a follow-up ticket in the DS and we move forward with this.